modal/runner/shared/download.py

```diff
@@ -22,6 +22,13 @@ def download_models(all_models: List[str]):
 
     cache_path = get_model_path("__cache__")
     for model_name in all_models:
+        parts = model_name.split(":")
+
+        model_revision = None
```

Comment:
This only downloads the model - can we also added it somehow into the protocol's input params?